### DISCUSSION TITLE
End of Life

### DISCUSSION THREAD

**Discussion by nathansb2022** [rancher/rke2#2823] [Q&A] [answered]
Is there an End of Life/End of Support document or web page? I am particularly looking at rke2 version: v1.20.15+rke2r2.

**Comment by brandond** [comment, marked as answer]
As an open-source project, we support things for the same period as upstream: https://kubernetes.io/releases/ . Accordingly, we stopped supporting the 1.20 branch when v1.23.0 was released.

If you have a support contract with SUSE, I would defer to the terms of your support contract, but in general the terms are outlined here: https://www.suse.com/lifecycle
